### ISSUE TITLE
Discrepancy between `tests/run-make/translation` test comment versus actual behavior

### ISSUE THREAD

**Issue by jieyouxu** [rust-lang/rust#135817]
https://github.com/rust-lang/rust/blob/ebbe63891f1fae21734cb97f2f863b08b1d44bf8/tests/run-make/translation/Makefile has a test comment and test case:

https://github.com/rust-lang/rust/blob/ebbe63891f1fae21734cb97f2f863b08b1d44bf8/tests/run-make/translation/Makefile#L23-L26

Ignoring that this is using the wrong `.ftl` file (it should be using `broken.ftl` to correspond to the test comment, where `broken.ftl` contains a slug that has a stray fluent interpolation argument), this must have regressed over the years because

```rs
rustc test.rs -Ztranslate-additional-ftl=broken.ftl
```

actually ICEs locally due to

```
thread 'rustc' panicked at compiler/rustc_errors/src/emitter.rs:1465:84:
called `Result::unwrap()` on an `Err` value: failed while formatting fluent string `parse_struct_literal_body_without_path`:
the fluent string has an argument `foo` that was not found.
help: no arguments are available
```

Pinging back to #132181.

I assume this is not the intended behavor.

**Comment by jieyouxu** [comment]
The Makefile test didn't catch it because it didn't set

```
SHELL=/bin/bash -o pipefail
```

**Comment by jieyouxu** [comment]
What's stranger is that this triggers an early ICE on Windows and Linux about the missing fluent interpolation variable, but not `aarch64-apple-darwin` because reasons?

**Comment by cyrgani** [comment]
closing as `-Ztranslate-additional-ftl` was removed by https://github.com/rust-lang/compiler-team/issues/967 and https://github.com/rust-lang/rust/pull/152455
